python/ray/tests/test_multi_node.py

```diff
@@ -448,6 +450,31 @@ def g():
         # Make sure the first driver ran to completion.
         assert "success" in out
 
+    nonexistent_id_bytes = _random_string()
+    # Define a driver that creates one task that depends on a nonexistent
+    # object. This task will be queued as waiting to execute.
+    driver_script = """
+import time
+import ray
+ray.init(redis_address="{}")
+@ray.remote
+def g(x):
+    return
+g.remote(ray.ObjectID({}))
+time.sleep(1)
+print("success")
+""".format(redis_address, nonexistent_id_bytes)
```

Comment:
This is failing in Travis. At least `"{}".format(ray.utils._random_string()).encode("ascii")` fails in Python 2. I'll try a quick fix.